docs/components_page/components/modal/scrollable.py

```diff
@@ -0,0 +1,55 @@
+import dash_bootstrap_components as dbc
+import dash_html_components as html
+from dash.dependencies import Input, Output, State
+
+modal = html.Div(
+    [
+        dbc.Button("Scrolling modal", id="open-scroll", className="mr-1"),
+        dbc.Button("Modal with scrollable body", id="open-body-scroll"),
+        dbc.Modal(
+            [
+                dbc.ModalHeader("Header"),
+                dbc.ModalBody(...),
```

Comment:
that doesn't look right?

Reply:
ah never mind -- this isn't actually run.

Reply:
Out of interest, since we're just using the variable LOREM, what's the rationale for having this file separately?

Reply:
That's a fair point. I didn't want to include the text in the example of course, but I guess injecting the variable `LOREM` into the example is no less clear than the ellipsis and means we only have one copy of the file. Is that what you had in mind?

Reply:
yes, exactly